openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TestCaseResolutionStatusRepository.java

```diff
@@ -96,121 +97,205 @@ private void validatePatchFields(
     }
   }
 
-  @Override
-  protected void postCreate(TestCaseResolutionStatus entity) {
-    super.postCreate(entity);
-    if (entity.getTestCaseResolutionStatusType() == TestCaseResolutionStatusTypes.Assigned) {
-      createAssignedTask(entity);
+  public Boolean unresolvedIncident(TestCaseResolutionStatus incident) {
+    return incident != null
+        && incident.getTestCaseResolutionStatusType() != TestCaseResolutionStatusTypes.Resolved;
+  }
+
+  private Thread getIncidentTask(TestCaseResolutionStatus incident) {
+    // Fetch the latest task (which comes from the NEW state) and close it
+    String jsonThread =
+        Entity.getCollectionDAO()
+            .feedDAO()
+            .fetchThreadByTestCaseResolutionStatusId(incident.getStateId());
```

Comment:
I hope we are encoding this in entityLink such that we can retrieve all the incidents(tasks) that belong to a test case along with a particular incident